google-cloud-asset/MIGRATING.md

```diff
@@ -29,6 +29,10 @@ To summarize:
     resource paths. These paths are now instance methods on the client objects,
     and are also available in a separate paths module. See
     [Resource Path Helpers](#resource-path-helpers) for more info.
+ *  Previously, RPC exceptions were subclasses of `Google::Gax::GaxError`,
+    defined in the `google-gax` gem. Now, RPSs throw exceptions subclassing
```

Comment:
Now that the client no longer uses `google-gax`, RPCs throw...

Reply:
Updated to remove the references to the gems (which is probably not necessary for the summary.)